stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/PollMessageContent.kt

```diff
@@ -115,10 +119,9 @@ public fun PollMessageContent(
             true -> ChatTheme.ownMessageTheme.deletedBackgroundColor
             else -> ChatTheme.otherMessageTheme.deletedBackgroundColor
         }
-
         else -> when (ownsMessage) {
-            true -> ChatTheme.colors.linkBackground
-            else -> ChatTheme.otherMessageTheme.backgroundColor
+            true -> ChatTheme.ownMessageTheme.poll.backgroundColor
+            else -> ChatTheme.otherMessageTheme.poll.backgroundColor
```

Comment:
Replaced with `PollStyle.backgroundColor` to avoid unexpected color changes for polls when there is a change for `linkBackground` for instance.